examples/releaseAllUsers/releaseallusers.go

```diff
@@ -21,6 +17,26 @@ const (
 // users in your account.
 //
 func main() {
+	var apiKey string
+	var urlBase string
+
+	// Obtain the input parameters
+	flag.StringVar(&urlBase, "url", "", "-url=<jumpcloud-api-url>")
+	flag.StringVar(&apiKey, "key", "", "-key=<API-key-value>")
+	flag.Parse()
+
+	if apiKey == "" {
+		fmt.Println("Usage of ./releaseAllUsers:")
+		fmt.Println("  -key=\"\": -key=<API-key-value>")
+		fmt.Println("  -url=\"\": -url=<jumpcloud-api-url>")
```

Comment:
I would probably hide this part, or let it be known that it is optional since we fallback to our prod env.